### PR TITLE
feat: add mandarin string literals

### PR DESCRIPTION
**Pull request by fewwwww**

### CODE REVIEW

**Comment by thiagocarvalhodev** [comment]
Thanks for the really hard work! We appreciate! 

As I'm unable to understand Mandarin, I'll chat with the team for help. I will give you an update soon!

**Comment by matibat** [comment]
TODO here
- Add 'zh' to the available locales

**Comment by thiagocarvalhodev** [comment]
Hey, @fewwwww ! We are following our internalization process with your translations. We'll keep you up to date with the progress. Thanks!

File: lib/l10n/app_zh.arb [hunk 1]
```diff
@@ -0,0 +1,1259 @@
...
+    "description": "Warning for the user that is about to share a drive"
+  },
+  "doneEmphasized": "完成",
+  "@doneEmphaseized": {
+    "description": "Done, no more to do here. Emphasized with upper case"
+  },
+  "errorLog": "错误日志",
+  "@errorLog": {
+    "description": "Error dialog"
+  },
```
Comment (matibat) [inline lib/l10n/app_zh.arb:655]:
```suggestion
```

File: lib/l10n/app_zh.arb [hunk 2]
```diff
@@ -0,0 +1,1259 @@
...
+    "description": "Uploading may take a while"
+  },
+  "failedToPrepareFileUpload": "准备上传文件时发生了错误",
+  "@failedToPrepareFileUpload": {
+    "description": "Error message for when it failed to prepare the upload"
+  },
+  "failedToPrepareFileUploadExplanation": "准备上传文件时发生了错误. 请使用桌面客户端, 或者尝试上传更小的文件.",
+  "@failedToPrepareFileUploadExplanation": {
+    "description": "Error message for when it failed to prepare the upload"
+  },
```
Comment (matibat) [inline lib/l10n/app_zh.arb:910]:
```suggestion
```

**Review by matibat — CHANGES_REQUESTED** [review]
Hey @fewwwww! Thanks again for the translations, we appreciate your contribution ❤️ 

We are making some cleanup in the codebase to remove unused text from the resource file, would you mind applying the suggestions I made here? Those are supposed to remove the following keys at the ARB file: `errorLog`, `failedToPrepareFileUpload`, and `failedToPrepareFileUploadExplanation`.

**Comment by fewwwww** [comment]
> Hey @fewwwww! Thanks again for the translations, we appreciate your contribution ❤️
> 
> We are making some cleanup in the codebase to remove unused text from the resource file, would you mind applying the suggestions I made here? Those are supposed to remove the following keys at the ARB file: `errorLog`, `failedToPrepareFileUpload`, and `failedToPrepareFileUploadExplanation`.

Done!

**Review by matibat — APPROVED** [review]
It looks good to me. Thank you! 🚀

**Pull request merged**